server/routes/api/documents/documents.ts

```diff
@@ -1975,7 +1977,9 @@ router.post(
       paranoid: false,
     });
 
-    await documentPermanentDeleter(documents);
+    await EmptyTrashTask.schedule({
```

Comment:
Oh, lets add a guard here for `documents.length > 0`

Reply:
We have one in the task.. - but probably safer (and efficient) to have it at both the places 👍

Reply:
We can avoid creating the event too if no work is being done